src/rabbitmq/Dockerfile

```
@@ -0,0 +1,17 @@
+FROM rabbitmq:latest
+
+WORKDIR /rabbitmq_data
+
+RUN rabbitmq-plugins enable rabbitmq_management
+RUN rabbitmq-plugins enable rabbitmq_federation
+RUN rabbitmq-plugins enable rabbitmq_federation_management
+
+ADD rabbitmq.config /etc/rabbitmq/
+ADD definitions.json /etc/rabbitmq/
+
+RUN chown rabbitmq:rabbitmq /etc/rabbitmq/rabbitmq.config /etc/rabbitmq/definitions.json
+
+CMD ["rabbitmq-server"]
+
+EXPOSE 5672
+EXPOSE 15672
```

Styling rules indicate to add blank space at the end of the file
```suggestion
EXPOSE 15672

```